project/docs/Makefile.jinja

```diff
@@ -0,0 +1,22 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
```

Comment:
What's the rationale for only setting SPHINXOPTS and SPHINXBUILD if it is undefined? In other words, why do you not want the Makefile to overwrite these env vars?

Reply:
I mainly followed sphinx quick-start template, which uses `?=` syntax, while [volttron docs](https://github.com/VOLTTRON/volttron/blob/main/docs/Makefile) uses `=` syntax this case.
According to the following references [GNU make](https://www.gnu.org/s/make/manual/make.html), [Overwrite variable from makefile](https://stackoverflow.com/questions/29434727/overwrite-variable-from-makefile), and [sphinx-build](https://www.sphinx-doc.org/en/master/man/sphinx-build.html)
- both syntax can be overridden by command line variable 
- `?=` is considered to be more persistent since it take into account environment variable as well. (Though I don't think we heavily rely on this feature.)